ci-operator/step-registry/ipi/conf/alibabacloud/cloud-creds-deprovision/ipi-conf-alibabacloud-cloud-creds-deprovision-ref.yaml

```diff
@@ -0,0 +1,9 @@
+ref:
+  as: ipi-conf-alibabacloud-cloud-creds-deprovision
+  from: cli
+  commands: ipi-conf-alibabacloud-cloud-creds-deprovision-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  documentation: The IPI cloud-creds-deprovision configure step adds a step that removes cloud access keys
```

Comment:
This and the provision step should document the `CREDENTIALSREQUEST_*` convention, and ideally give some background around why `ccoctl` is unable to reap these creds without that content.  I'd have naively expected a common `cluster_id` to have been enough.  Will we be doc'ing customers needing to preserve these somewhere outside the cluster so they can delete their creds when they're done with the cluster?

Reply:
Added documentation.

Either the preservation or the regeneration will be required as the code is written today. We could file an issue against CCO repository and request that an Alibaba engineer see if it is possible to remove by using the `cluster_id`.

Reply:
Added https://github.com/openshift/cloud-credential-operator/issues/437